10-zwave.js

```diff
@@ -410,7 +410,16 @@ module.exports = function(RED) {
 					) {
 						if (debug) console.log('attempting direct call to OpenZWave API: %s(%s)', msg.topic, payload);
 						try {
-							ozwDriver[msg.topic].apply(ozwDriver, payload);
+							var retval = ozwDriver[msg.topic].apply(ozwDriver, payload);
+							var newmsg = {'topic': msg.topic};
+							if (retval) {
+								if (debug) console.log('Got return value, sending as payload');
+								newmsg.payload = retval;
+							} else {
+								if (debug) console.log('No return value but call successful, sending empty array');
+								newmsg.payload = [];
+							}
+							node.send(newmsg);
```

Comment:
It would attempt to send, but effectively do nothing since there's no output wires on the zwave-out node. Tweaking the zwave-out node's behavior was the lowest impact way to just glue the functionality on without stripping experimental API support from the zwave-out node (yet).

I feel like long term, we'd want to split them out - with the new node type handling all the direct API stuff, zwave-out handing what it's always handled, and with the them both having different constructors.